docs/docs/configuration/provisioning.md

```diff
@@ -0,0 +1,50 @@
+# Provisioning server
+
+Tracetest allows a server to be provisioned the first time a new Tracetest server is installed and launched. Provisioning sets certain resources in the server to the specified values, allowing you to configure the server. It is convenient in a CI/CD flow where you want to launch a server with a specified configuration. 
```

Comment:
```suggestion
Tracetest allows a server to be provisioned the first time Tracetest is installed and launched. Provisioning sets certain resources in the server to the specified values, allowing you to configure the server. It is convenient in a CI/CD flow where you want to launch a server with a specified configuration. 
```

Reply:
I think I may want to leave this wording to distinguish between a server install and the CLI install. Maybe it is not needed???